lib/project.rb

```
@@ -311,5 +315,20 @@ def self.generate_cache_configs(config)
         end
       end
     end
+
+    def self.resolve_extension_path(ext, php_version)
```

RuboCop (1.84.1)

[warning] `private` (on line 213) does not make singleton methods private. Use `private_class_method` or `private` inside a `class << self` block instead.

(Lint/IneffectiveAccessModifier)